R/tt_toString.R

```diff
@@ -42,9 +37,10 @@ NULL
 #' tbl <- build_table(lyt, iris2)
 #'
 #' cat(toString(tbl, col_gap = 3))
+#'
 #' @rdname tostring
-#' @name tostring
-#' @aliases toString,VTableTree-method
+#' @aliases tostring toString,VTableTree-method
+#' @exportMethod toString
```

Comment:
was this source of issues??

Reply:
Yes it was causing some issues wiht inherited documentation in other functions.

Reply:
Was it the part of the usual warnings? If you manage to fix those it would be amazing